packages/driver/src/cy/commands/connectors.coffee

```diff
@@ -179,7 +179,8 @@ module.exports = (Commands, Cypress, cy, state, config) ->
         consoleProps: ->
           Subject: subject
 
-    if not str
+    ## check for false positive (negative?) with 0 given as index
+    if not str and str isnt 0
```

Comment:
since this logic is shared between line 256 and this line, move it to a named function for clarity. maybe something like this:

```coffee
isObjectKey = (i) =>
	i or i is 0
```

Reply:
Done